packages/fluentui/react-northstar/src/themes/teams/components/Dialog/dialogVariables.ts

```diff
@@ -44,4 +47,6 @@ export const dialogVariables = (siteVariables): Partial<DialogVariables> => ({
   overlayZIndex: siteVariables.zIndexes.overlay,
 
   headerActionMargin: `${pxToRem(-3)} ${pxToRem(-8)} 0 0`,
+  // in case of customized footer this var allows to align styles
+  footerActionsBreakpoint: '400px',
```

Comment:
maybe, I just don't understand enough about TMP styles 😅 do we expect this to be used in theme overrides ?